content/applications/marketing/marketing_automation/advanced/understanding_metrics.rst

```diff
@@ -1,80 +1,173 @@
 
-.. note::
-   The filter here is applied to *all* activities. Leads that lose that specific :guilabel:`Tag`
-   in the meantime will be excluded from later activities.
+Beside that, there is also the option to view the statistics as :guilabel:`Stacked`, and the data
+can be put into :guilabel:`Descending` or :guilabel:`Ascending` order.
 
-.. tip::
-   Filters can also be applied to individual activities, in the :guilabel:`Domain` section of the
-   activity pop-up form. Use this feature to specify an individual filter that will only be
-   performed if the records satisfy *both* filter criteria - the activity filter and its specific
-   domain filter.
+To the far-left of the view options, there is the :guilabel:`Measures` drop-down menu. When clicked,
+the options to view the :guilabel:`Number of Clicks` or total :guilabel:`Count` are available. And,
+to the right of the :guilabel:`Measures` drop-down menu, there's the ability to add any data to a
+spreadsheet by clicking the :guilabel:`Insert in Spreadsheet` button.
 
-   (*Example*: when configuring an SMS activity, make sure a certain phone number is set,
-   in order to avoid triggering a SMS that would never be sent.)
+Also, in the upper-right corner of the :guilabel:`Link Statistics` page, to the far-right of the
+search bar, there are additional view options to choose from: the default :guilabel:`Graph` view,
+the :guilabel:`Pivot` table view, and the :guilabel:`List` view.
 
-   .. image:: understanding_metrics/individual-filter-to-activity.png
-      :align: center
-      :alt: A relevant targeting filter that applies to a type of individual activity.
+Traces
+======
 
-:guilabel:`Success` represents the number of times the searching for participants - who match the
-criteria filter(s) of that activity - was performed successfully, in relation to the total number
-of participants. If a participant does *not* match the conditions, it will be added to
-:guilabel:`Rejected`.
+Odoo tracks all activities used in every marketing campaign. The data related to these activities
+can be accessed and analyzed in the :guilabel:`Traces` page, which can be found by navigating to
+:menuselection:`Marketing Automation app --> Reporting --> Traces`.
 
-.. image:: understanding_metrics/participant-matching-success-vs-rejected.png
+.. image:: understanding_metrics/traces-page-marketing-automation.png
    :align: center
-   :alt: Overview showing participants who do or do not match filtering criteria on an activity.
+   :alt: The Traces page in the Odoo Marketing Automation application.
 
-Hovering over the graph, Odoo displays the number of successful and rejected participants, per day,
-over the last 15 days.
+The default view on the :guilabel:`Traces` page is the :guilabel:`Bar Chart` view, but there are
+different view options available in the upper-left corner. There is the option to view the
+statistics as a :guilabel:`Line Chart` or :guilabel:`Pie Chart`.
```

Comment:
Although this paragraph isn't wrong, I felt like I was reading a description of the UI instead of about the function of the feature. I tried to describe what I think I'm seeing on this Link Statistics page in my own words, but I'm not really sure... 

Feel free to correct anything!

```suggestion
On the :guilabel:`Link Statistics` page, click-through engagement with each activity in the
marketing campaign is displayed in the :guilabel:`Bar Chart` view, by default. Alternative options
to view this data in a :guilabel:`Line Chart` or :guilabel:`Pie Chart` is available in the
upper-left corner.
```